server/src/main/java/org/apache/iotdb/db/qp/Planner.java

```diff
@@ -65,6 +72,46 @@ public PhysicalPlan parseSQLToPhysicalPlan(String sqlStr, ZoneId zoneId)
     return physicalGenerator.transformToPhysicalPlan(operator);
   }
 
+  /**
+   * convert raw data query to physical plan directly
+   */
+  public PhysicalPlan rawDataQueryToPhysicalPlan(List<String> paths, long startTime, long endTime)
```

Comment:
rawDataQueryToPhysicalPlan  ----> rawDataQueryReqToPhysicalPlan
and use the class rawDataQueryReq as parameter, maybe we will support more complicated query req

Reply:
Got it, thank you Chao for your comments!